src/pages/Mint.js

```diff
@@ -11,7 +11,12 @@ const LOCAL_STORAGE_KEY = 'storiesApp.stories'
 // TODO how do we get the storyRef here, and move handleAddStory here
 export default function Mint({storiesState}) {
 
-  const [stories, setStories] = storiesState // state passed in
```

Comment:
don't think we should delete this yet, isn't it watching for a change in storiesState

Reply:
Yeah you are right, lets keep it in.